vpr/src/place/move_utils.h

```diff
@@ -59,4 +101,15 @@ bool placer_breakpoint_reached();
 // Should be setted in the breakpoint calculation algorithm
 void set_placer_breakpoint_reached(bool);
 
+bool find_to_loc_median(t_logical_block_type_ptr type, const t_bb* limit_coords, const t_pl_loc from, t_pl_loc& to);
```

Comment:
Should move the comment of how to call these functions / outline of what they do to here. Convention is to comment how to use something in the header, details of how it works in the .cpp

Reply:
Fixed